sdk/keyvault/azure-mgmt-keyvault-autorestv5/azure/mgmt/keyvault/v2018_02_14/models/_models.py

```diff
@@ -547,23 +548,29 @@ def __init__(
         super(PrivateEndpointConnection, self).__init__(**kwargs)
         self.private_endpoint = kwargs.get('private_endpoint', None)
         self.private_link_service_connection_state = kwargs.get('private_link_service_connection_state', None)
-        self.provisioning_state = kwargs.get('provisioning_state', None)
+        self.provisioning_state = None
 
 
 class PrivateEndpointConnectionItem(msrest.serialization.Model):
     """Private endpoint connection item.
 
+    Variables are only populated by the server, and will be ignored when sending a request.
+
     :param private_endpoint: Properties of the private endpoint object.
     :type private_endpoint: ~azure.mgmt.keyvault.v2018_02_14.models.PrivateEndpoint
     :param private_link_service_connection_state: Approval state of the private link connection.
     :type private_link_service_connection_state:
      ~azure.mgmt.keyvault.v2018_02_14.models.PrivateLinkServiceConnectionState
-    :param provisioning_state: Provisioning state of the private endpoint connection. Possible
+    :ivar provisioning_state: Provisioning state of the private endpoint connection. Possible
```

Comment:
Did anything change in the swagger in your regeneration for `provisioning_state` to become readonly?

Reply:
> Did anything change in the swagger in your regeneration for `provisioning_state` to become readonly?

@iscai-msft I didn't change anything on swagger, is this an unexpected change?

Reply:
Ok I see, this is my bad. We made a fix where readonly being defined in the object definition propagates up so the property becomes readonly as well. Sorry for the confusion